source/agora/registry/API.d

```diff
@@ -43,6 +47,25 @@ public struct RegistryPayload
     /// signature over the `data` member
     public Signature signature;
 
+    /***************************************************************************
+
+        make a key for a `RegistryPayload`
+
+        Params:
+            pubkey = the address used to make a key for a `RegistryPayload`
+            utxo = the UTXO used to make a key for a `RegistryPayload`
+
+        Returns:
+            a key of `Hash` type
+
+    ***************************************************************************/
+
+    static public Hash makeRegistryKey (in PublicKey pubkey, in UTXO utxo)
```

Comment:
do we really need to has the pubkey and utxo together? utxo has an output, and if the utxo is a payment transaction (as opposed to be a script), then the output should contain the pubkey(or its hash)

Reply:
Good point! But there are some cases where`outsider_validators` don't yet have UTXOs in unit tests. I will consider another way to deal with it. Ty.